packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -738,6 +739,10 @@ export default class BackupNg {
       }
     }
 
+    if (!some(vm.$VBDs, { is_cd_drive: false })) {
```

Comment:
There is no such property on a [XAPI VBD record](https://xapi-project.github.io/xen-api/classes/vbd.html).